local-cli/runAndroid/runAndroid.js

```diff
@@ -92,17 +92,20 @@ function buildAndRun(args, reject) {
   }
 
   try {
-    const packageName = fs.readFileSync(
-      'app/src/main/AndroidManifest.xml',
+    const manifest = fs.readFileSync(
+      'app/build/intermediates/manifests/full/debug/AndroidManifest.xml',
```

Comment:
Since we can customized the build folder, it's not always `app/build` (we've this in our docs too, and should support this use case).